.eslintrc

```diff
@@ -59,6 +59,7 @@ rules:
   indent: [2, 2, {SwitchCase: 1}]
   key-spacing: [2, {mode: "minimum"}]
   keyword-spacing: 2
+  linebreak-style: 2
```

Comment:
Yes, please be explicit. If they ever change the default, we'd be screwed. :laughing: